app/models/nic/interface.rb

```diff
@@ -20,10 +20,28 @@ class Interface < Base
 
     validate :alias_subnet
 
-    delegate :network, :to => :subnet
+    delegate :network, :to => :subnet, :prefix => true
+    delegate :network, :to => :subnet6, :prefix => true
```

Comment:
The `:prefix => true` changes the `delegate` behaviour: instead of a method `network` pointing to `subnet.network`, it creates a method `subnet_network` (still pointing to `subnet.network`). The same is done for `subnet6`: a method `subnet6_network` pointing to `subnet6.network`